apps/graphql/src/apps/itinerary/queries/__tests__/__fixtures__/checkItinerary/CheckItinerary.graphql

```diff
@@ -0,0 +1,33 @@
+fragment RouteStopFragment on RouteStop {
+  time {
+    utc
+    local
+  }
+}
+
+{
+  checkItinerary(input: { bookingToken: "token", bags: 1, passengers: { adults: 1 }}) {
+    id
+    valid
+    checked
+    sectors {
```

Comment:
Why do you include `sectors` in this query?

Reply:
Same test query like for `searchItineraries` with sectors is used because we expect basically same data, just with extra isChecked and isValid. Because we are also displaying whole itinerary info on the result/booking detail page.

Reply:
I just see it's resolved as `[]` in the snapshot so it seems useless here.

Reply:
Data will be mapped to sectors in separate PR. But you are right, it can be confusing to have it there now.
So I removed sectors and will update it once there will be some reasonable content for snapshots.